### PR TITLE
Added info for HeatIt Scene Controllers

### PR DESCRIPTION
**Pull request by networkpotato**
Added info for the wall switches/scene controllers from HeatIt/ThermoFloor:

https://www.heatit.com/wp-content/uploads/2018/11/Heatit-Z-Push-Button-2.pdf
https://www.heatit.com/wp-content/uploads/2018/11/Heatit-Z-Push-Button-8.pdf

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_docs/z-wave/device-specific.markdown [hunk 1]
```diff
@@ -614,4 +614,46 @@ switch:
...
+### {% linkable_title HeatIt/ThermoFloor Z-Push Button 2/8 Wall Switch %}
+
+To get the Z-Push Button 2 or the Z-Push Button 8 working in Home Assistant, you must first edit the `COMMAND_CLASS_CENTRAL_SCENE` in your `zwcfg` file.
+
+1. Go the Z-Wave control panel in Home Assistant and make a note of the node number your wall switch has been assigned.
+2. *Stop* Home Assistant.
+3. Make a backup of your `zwfcg` file, just in case.
+4. In the `zwcfg` file, find the `Node id` that corresponds to the number you noted in the first step.
+5. Within the `Node id` you identified, highlight everything between `<CommandClass id="91"` and `</CommandClass>` (inclusive) and paste in the following:
+5.1. For the Z-Push Button 2:
```
Comment (frenck) [inline source/_docs/z-wave/device-specific.markdown]: This is not working that way in Markdown. Please indent the sub list.

File: source/_docs/z-wave/device-specific.markdown [hunk 2]
```diff
@@ -614,4 +614,46 @@ switch:
...
+
+```xml
+    <CommandClass id="91" name="COMMAND_CLASS_CENTRAL_SCENE" version="1" request_flags="4" innif="true" scenecount="0">				<Instance index="1" />
+	<Value type="int" genre="system" instance="1" index="0" label="Scene Count" units="" read_only="true" write_only="false" verify_changes="false" poll_intensity="0" min="-2147483648" max="2147483647" value="0" />
+	<Value type="int" genre="user" instance="1" index="1" label="Button 1" units="" read_only="true" write_only="false" verify_changes="false" poll_intensity="0" min="-2147483648" max="2147483647" value="0" />
+	<Value type="int" genre="user" instance="1" index="2" label="Button 2" units="" read_only="true" write_only="false" verify_changes="false" poll_intensity="0" min="-2147483648" max="2147483647" value="0" />
+    </CommandClass>
+```
+
+5.2. For the Z-Push Button 8:
```
Comment (frenck) [inline source/_docs/z-wave/device-specific.markdown]: This is not working that way in Markdown. Please indent the sub list.

File: source/_docs/z-wave/device-specific.markdown [hunk 3]
```diff
@@ -614,4 +614,46 @@ switch:
...
+To get the Z-Push Button 2 or the Z-Push Button 8 working in Home Assistant, you must first edit the `COMMAND_CLASS_CENTRAL_SCENE` in your `zwcfg` file.
+
+1. Go the Z-Wave control panel in Home Assistant and make a note of the node number your wall switch has been assigned.
+2. *Stop* Home Assistant.
+3. Make a backup of your `zwfcg` file, just in case.
+4. In the `zwcfg` file, find the `Node id` that corresponds to the number you noted in the first step.
+5. Within the `Node id` you identified, highlight everything between `<CommandClass id="91"` and `</CommandClass>` (inclusive) and paste in the following:
+5.1. For the Z-Push Button 2:
+
+```xml
```
Comment (frenck) [inline source/_docs/z-wave/device-specific.markdown]: In order not to break the Markdown numbering, you need to indent this code block at least 4 spaces relative to the list that it is in.

File: source/_docs/z-wave/device-specific.markdown [hunk 4]
```diff
@@ -614,4 +614,46 @@ switch:
...
+    <CommandClass id="91" name="COMMAND_CLASS_CENTRAL_SCENE" version="1" request_flags="4" innif="true" scenecount="0">				<Instance index="1" />
+	<Value type="int" genre="system" instance="1" index="0" label="Scene Count" units="" read_only="true" write_only="false" verify_changes="false" poll_intensity="0" min="-2147483648" max="2147483647" value="0" />
+	<Value type="int" genre="user" instance="1" index="1" label="Button 1" units="" read_only="true" write_only="false" verify_changes="false" poll_intensity="0" min="-2147483648" max="2147483647" value="0" />
+	<Value type="int" genre="user" instance="1" index="2" label="Button 2" units="" read_only="true" write_only="false" verify_changes="false" poll_intensity="0" min="-2147483648" max="2147483647" value="0" />
+    </CommandClass>
+```
+
+5.2. For the Z-Push Button 8:
+
+```xml
```
Comment (frenck) [inline source/_docs/z-wave/device-specific.markdown]: In order not to break the Markdown numbering, you need to indent this code block at least 4 spaces relative to the list that it is in.

**Review by frenck — DISMISSED** [review]

**Review by klaasnicolaas — APPROVED** [review]

**Pull request merged**